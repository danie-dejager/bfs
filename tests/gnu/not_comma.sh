# Regression test: assertion failure in sink_not_comma()
bfs_diff basic -not \( -print , -name '*f*' \) -print
