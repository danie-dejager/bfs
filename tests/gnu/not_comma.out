basic
basic
basic/a
basic/a
basic/b
basic/b
basic/c
basic/c
basic/c/d
basic/c/d
basic/e
basic/e
basic/e/f
basic/g
basic/g
basic/g/h
basic/g/h
basic/i
basic/i
basic/j
basic/j
basic/j/foo
basic/k
basic/k
basic/k/foo
basic/k/foo/bar
basic/k/foo/bar
basic/l
basic/l
basic/l/foo
basic/l/foo/bar
basic/l/foo/bar
basic/l/foo/bar/baz
basic/l/foo/bar/baz
